templates/tg-mini-app-boilerplate-template/README.md

```diff
@@ -0,0 +1,45 @@
+## Overview
+
+This template is made from the [boilerplate template](https://github.com/aptos-labs/create-aptos-dapp/tree/main/templates/boilerplate-template), the difference is that we make this possible to run as a Telegram mini app, and we change the app to be a simple clicker game.
```

Comment:
I dont think it is relevant to mention the Boilerplate template as it is a clocker game and not a boilerplate. Lets just write on this template and what is provides